recipe/meta.yaml

```diff
@@ -119,8 +121,10 @@ requirements:
     - mkl {{ mkl }}.*                 # [blas_impl == "mkl"]
     - openblas                        # [blas_impl == "openblas"]
     # OpenMP
-    - intel-openmp                    # [(osx or win) and x86]
-    - llvm-openmp                     # [osx and arm64]
+    # We pull in the same versions of mkl and intel-openmp: intel aligns the versions
+    # We use intel-openmp for all mkl variants.
+    # For openblas on win and linux, we don't specify any openmp implementation; it comes from the compiler.
+    - intel-openmp   {{ mkl }}        # [blas_impl == "mkl"]
```

Comment:
libprotobuf needs to be pinned in host to version 3.20.1

Reply:
additional check in host section what is the lowest compatible version of all un-pinned dependencies and add them (plus their corresponding run pinnings)